### PR TITLE
Added token handling section and diagram

### PR DESCRIPTION
**Pull request by pareek-narendra**
#### Checklist

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/developer-authentication-demos/blob/master/guidelines.md) before creating this PR.
- [x] This is not a duplicate entry for any framework or language\

##### Description of change

Please provide a small description of what you did.

Added more details in guideline along with diagrams and steps.

### CODE REVIEW

File: guidelines.md [hunk 1]
```diff
@@ -20,28 +21,75 @@ API credentials are as below
 Create a project under your tech stack folder. 
 Use Existing Login/Register page and profile page available in the repository
 
-**Note**: Get refrence from sample demo of LoginRadius IAM implementation using [Node.js and Express framework](/demos/nodejs/express/readme.md) under demos -> nodejs -> express.
+### Frontend
+
+Checkout the [Developer demo theams](/demos/theme/readme.md) for Folder staructure and how to utilize and work with token on client side.
```
Comment (vyasriday) [inline guidelines.md:26]: Spelling mistakes for themes, structure.

File: guidelines.md [hunk 2]
```diff
@@ -20,28 +21,75 @@ API credentials are as below
...
+    |----toast.css
+  --|index.html
+  --|profile.html
+  --|readme.md
+
+```
+
+### Backend
+
+Get refrence from sample demo of LoginRadius IAM implementation using [Node.js and Express framework](/demos/nodejs/express/readme.md) under demos -> nodejs -> express.
```
Comment (vyasriday) [inline guidelines.md:55]: reference spelling mistake.

File: guidelines.md [hunk 3]
```diff
@@ -20,28 +21,75 @@ API credentials are as below
...
+### Handling token
+
+Look into this diagram to understand the Authentication flow and token handling.
+
+![Image](auth_flow.png)
+
+In the Above diagram:
+- Your Application will redirect you to LoginRadius IDX page once you click on the Login Button.
+- Login and Authenitcate the User
+- IDX page will return you to the return_url along with the token
```
Comment (vyasriday) [inline guidelines.md:83]: wrap return_url in quotes like `return_url`

File: guidelines.md [hunk 4]
```diff
@@ -77,4 +125,24 @@ You can utilize our existing SDKs functions to handle access token
...
\ No newline at end of file
+![Image](support_ticket.png)
+
+
+
+## FAQs
+
+1. How we are reading the users Access Token?
+
+After successful login from LoginRadius IDX, the access token is returned as the query parameter to the callback URL, Access token is been read from the callback URL and stored in the browser.
```
Comment (vyasriday) [inline guidelines.md:136]: `Access token is been read from the callback URL` can be `Access token is read from the callback URL`

File: guidelines.md [hunk 5]
```diff
@@ -77,4 +125,24 @@ You can utilize our existing SDKs functions to handle access token
...
+
+
+2. How logout is working if we do logout on IDX page?
+
+After Logout action is performed on the IDX successfuly, user is redirected to the callback URL. On the callback page, we are checking if the access token validity. If invalid, Access Token is been removed from the localStorage and Cookies.
```
Comment (vyasriday) [inline guidelines.md:141]: change to `After Logout action is performed on the IDX successfully, the user is redirected to the callback URL. On the callback page, we are checking if the access token is valid. If invalid, Access Token is removed from the localStorage and Cookies. `

File: guidelines.md [hunk 6]
```diff
@@ -77,4 +125,24 @@ You can utilize our existing SDKs functions to handle access token
...
+
+After successful login from LoginRadius IDX, the access token is returned as the query parameter to the callback URL, Access token is been read from the callback URL and stored in the browser.
+
+
+2. How logout is working if we do logout on IDX page?
+
+After Logout action is performed on the IDX successfuly, user is redirected to the callback URL. On the callback page, we are checking if the access token validity. If invalid, Access Token is been removed from the localStorage and Cookies. 
+
+
+3. How we are managing the Access Token on local and IDX?
```
Comment (vyasriday) [inline guidelines.md:144]: change to `How are we managing the Access Token on local and IDX?`

**Review by vyasriday — CHANGES_REQUESTED** [review]

**Comment by pareek-narendra** [comment]
@vyasriday 
Added the suggested changes and also added the video tutorial link for dev authentication implementation.

**Review by vyasriday — APPROVED** [review]

**Pull request merged**